integration-tests/src/fees.rs

```diff
@@ -40,10 +88,103 @@ fn nft_mint_transaction_fees() {
 		};
 
 		let info = call.get_dispatch_info();
-		let len = call.encoded_size() as u32;
-		let fees = TransactionPayment::compute_fee(len, &info, 0);
+		//rust encoded fees
+		let rust_encoded_len = call.encoded_size() as u32;
+		let rust_encoded_fees = TransactionPayment::compute_fee(rust_encoded_len, &info, 0);
+
+		//UI encoded fees
+		let ui_encoded_len = 192;
+		assert_eq!(ui_encoded_len - rust_encoded_len, 105);
```

Comment:
let's also add an assert message or comment?
something like "lengths expected to differ by a constant value at test creation, revisit if encoded call length changes")

Reply:
adjusted